lib/texting.rb

```diff
@@ -0,0 +1,30 @@
+require 'twilio-ruby'
+require 'dotenv'
+Dotenv.load
+
+module Texting
+
+  def self.send_text
+
+    account_sid = ENV['ACCOUNT_SID']
+    auth_token = ENV['AUTH_TOKEN']
+    from = ENV['TWIL_PHONE_NUMBER']
+    to = ENV['MY_PHONE_NUMBER']
+
+    delivery_time = Time.new + (60*60)
+    message = "Thank you! Your order has been placed and delivery " +
```

Comment:
Style/LineEndConcatenation: Use `\` instead of `+` or `<<` to concatenate those strings.